### PR TITLE
Create concept for the Ontohub 2.0 user interface

### PR DESCRIPTION
**Pull request by tom-krusch**

### CODE REVIEW

**Comment by tillmo** [comment]
before we approve this pull request, please first update the concept as discussed this morning

**Comment by tom-krusch** [comment]
this is the updated version pf the concept as discussed this morning

**Comment by eugenk** [comment]
Welcome to the team, @tom-krusch and @TimKrusch!

The `doc/` directory is not the right place for such a conceptual description. It's rather for technical documentation of the implementation. Please create a new [wiki page](https://github.com/ontohub/ontohub-frontend/wiki) instead and put your concept there. (We just opened the frontend wiki for your first tasks.)

It's a little bit difficult to follow the text without a picture of what you are describing. If you could additionally create pen-and-paper-prototypes of your concept and put pictures of them into the wiki, that would be great.

If you have any questions to your tasks, feel free to post them in the corresponding issue and we'll do our best to help you to fulfil your task.

**Comment by tom-krusch** [comment]
@eugenk, Wir werden unsere Dateien erstmal im "doc" Folder hochladen und das weitere Vorgehen mit @tillmo in unserem nächsten Treffen (am Dienstag) besprechen. Danke trotzdem für die Hinweise.

**Comment by tillmo** [comment]
errrors should be displayed similarly as in overleaf

**Comment by tom-krusch** [comment]
maybe we can discuss the errors-function in our meeting tomorrow.

**Comment by tillmo** [comment]
Landing page: 
* when logged out: featured repositories
* when logged in: recent changes, see github

**Comment by tillmo** [comment]
the file browser should be a bit more complex: repo -> folder -> file -> OMS/mapping

File: doc/concept for the Ontohub 2.0 user interface [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+- OMS Graph
+- Übersetzung
+
+Funktionen als PopUp:
+- Commit
+- erweiterte Suche mit Filter
+- Compare
+- Versionsmanagement/ Diff
+- schneller Zugriff auf Fehlermeldungen
+
```
Comment (tillmo) [inline doc/concept for the Ontohub 2.0 user interface]: as own panel: history
in global menu bar: search with context dependent scope (all repos, this repo, this file, this OMS)

general panel structure
file browser - content - (optional) command
file browser: standard file browser, graph
content: README.md, OMS details, editor, diff, translation, translation-compare
commands: prove, translate, git with staged files (each staged file with show diff cation)/commit message+button/revert/add to staged 
mobile version: only one panel is shown, tabs for switching between panels

multi-file commit: mark staged files in file browser

**Pull request closed without merging**